docs/dockerfile_experimental.md

```diff
@@ -0,0 +1,128 @@
+# Dockerfile frontend experimental syntaxes
+
+The contents of this document is planned to be moved to https://docs.docker.com in future.
+
+## Note for Docker users
+
+If you are using Docker v18.06 or later, BuildKit mode can be enabled by setting `export DOCKER_BUILDKIT=1` on the client side.
+Docker v18.06 also requires the daemon to be running as the experimental mode.
```

Comment:
Should be "...to be running in experimental mode.".

Also maybe a link to some doc? The "note" at https://docs.docker.com/engine/reference/commandline/dockerd/#description was the best I could find, which isn't ideal (perhaps someone knows a better one) but better than nothing.